### PR TITLE
Cleanup and fixes

### LINKED ISSUE #82: IP-Addresses optainable without confirmation
IP-Addresses can be optained from the wizard without validating the e-mail-address by this workflow:

1. Request IP-Address allocation (amateur mode)
2. do not validate the email address by not clicking on the link sent to you
3. request a list of all alloc'd IP-Addresses

In result, a user can get the addresses without validating the email address. The allocation will expire and the addresses will be alloc'd to another user and node. This leads to duplicate addresses, which is bad for the network.

### PR DESCRIPTION
**Pull request by nicolasberens**
- removes old routerdb code (#89)
- removes old wizard since the simplemode replaced it
- updates a few dependencies
- fixes #82

### CODE REVIEW

File: app/templates/wizard/email.txt
Comment (Akira25) [inline app/templates/wizard/email.txt:1]: Are you sure, this file should be deleted entirely? Does the server will still send emails then?

File: app/templates/wizard/form.html
Comment (Akira25) [inline app/templates/wizard/form.html:1]: This looks like the template for the input form in simple mode, isn't it?

**Review by Akira25 — COMMENTED** [review]
Besides the two templates, this looks good to me.

**Comment by Akira25** [comment]
Cool branch name, btw. xD

File: app/templates/wizard/form.html
Comment (nicolasberens) [reply app/templates/wizard/form.html:1]: no, those are here: https://github.com/freifunk-berlin/config.berlin.freifunk.net/tree/master/app/templates/simplemode

File: app/templates/wizard/email.txt
Comment (nicolasberens) [reply app/templates/wizard/email.txt:1]: yes

**Pull request merged**